engine/src/test/java/com/redhat/cloud/notifications/InventoryTestHelpers.java

```diff
@@ -80,8 +79,7 @@ public static EmailAggregation createEmailAggregation(String tenant, String bund
 
         emailActionMessage.setOrgId(DEFAULT_ORG_ID);
 
-        JsonObject payload = baseTransformer.toJsonObject(emailActionMessage);
-        aggregation.setPayload(payload);
+        aggregation.setPayload(TestHelpers.wrapActionToJsonObject(emailActionMessage));
```

Comment:
`baseTransformer` is no longer needed in this class.

Reply:
Removed!